src/app/counter-analytics/day-of-week-histogram/day-of-week-histogram.page.component.ts

```diff
@@ -0,0 +1,46 @@
+import { AfterViewInit, Component, ElementRef, OnInit, ViewChild } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { CounterAnalyticsService } from '../../services/counter-analytics.service';
+import 'anychart';
+import { Counter } from '../../models/counter';
+
+@Component({
+  selector: 'app-counter-analytics',
+  templateUrl: './day-of-week-histogram.page.component.html',
+  styleUrls: ['./day-of-week-histogram.page.component.scss'],
+})
+export class DayOfWeekHistogramPage implements OnInit, AfterViewInit {
+
+  counter: Counter;
+  chart: anychart.charts.Cartesian = anychart.column();
+
+  @ViewChild('chartContainer', {static: true}) container: ElementRef;
+
+  constructor(
+    private route: ActivatedRoute,
+    private counterAnalytics: CounterAnalyticsService
+  ) {}
+
+  ngOnInit() {
+    const weekDays = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
```

Comment:
European weeks start with Monday, maybe make this configurable in the settings page?